modules/installation-mirror-repository.adoc

```diff
@@ -21,13 +21,17 @@ endif::[]
 ifdef::openshift-origin[]
 * You have created a pull secret for your mirror repository.
 endif::[]
+* You generated self-signed certificate and trusted it.
 
-* If you use self-signed certificates that do not set a Subject Alternative Name, you must precede the `oc` commands in this procedure with `GODEBUG=x509ignoreCN=0`. If you do not set this variable, the `oc` commands will fail with the following error:
-+
-[source,terminal]
-----
-x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
-----
+[NOTE]
```

Comment:
I think most of the original sentence looks good. Only need to prompt user to make sure set a Subject Alternative Name in self-signed certificates. So something like:
```
* If you use self-signed certificates, you must set a Subject Alternative Name in self-signed certificates.
```

Reply:
addressed